src/containers/AsteroidClassContainer.jsx

```diff
@@ -193,16 +192,14 @@ class AsteroidClassContainer extends React.PureComponent {
     return (
       <ConditionalWrapper
         condition={multiple && !!overlayData}
-        wrapper={children => (
```

Comment:
Ah.  I understand your intuition here, but this is not quite right.  We actually still need to do this `children` magic in order for this ContionalWrapper component to work right.  It'll make more sense as you begin to work on Please revert to how it was previous [EPO-3932](https://jira.lsstcorp.org/browse/EPO-3932) next sprint (or you can do that ticket this sprint if you prefer).  Basically `children` is this like variable standing in for whatever CondiitonalWrapper is conditionally wrapping.  And the `wrapper` prop is maybe a bit misleadingly named as it's not JUST the component you want to be the wrapper, but is a function that returns the whole kit-and-kaboodle: the wrapper component and the children (the stuff being wrapped).  I'm confusing myself as I'm writing it.  Like I said we can chat about it more when you start working on EPO-3932.  For not change this line back to:
`wrapper={children => (`

and I'll indicate below how to change the next bit